### ISSUE TITLE
HTTPS not working on Debian 10

### ISSUE THREAD

**Issue by liszca** [fusiondirectory/fusiondirectory#53]
During the Setup I choose "Enforce encrypted connections" under "Login and session", but that gave me an wired error, so installed it without HTTPS and gave it another try by following this guide: 

https://www.digitalocean.com/community/tutorials/how-to-create-a-self-signed-ssl-certificate-for-apache-in-debian-10 

This didn't help:
```
Fatal error: Uncaught Error: Call to undefined function xml_parser_create() in /usr/share/fusiondirectory/include/class_config.inc:146 Stack trace: #0 /usr/share/fusiondirectory/include/class_config.inc(135): config->parse_data('<?xml version="...') #1 /usr/share/fusiondirectory/include/class_config.inc(83): config->parse('/etc/fusiondire...') #2 /usr/share/fusiondirectory/html/index.php(165): config->__construct('/etc/fusiondire...', '/usr/share/fusi...') #3 {main} thrown in /usr/share/fusiondirectory/include/class_config.inc on line 146
```

During the setup there are paths mentioned, "Key path", "Certificate path" and "CA certificate path", these are empty when I choose "Enforce encrypted connections", I guess that maybe the reason why it fails, but don't know how to fix.

My installation source:
```
deb http://repos.fusiondirectory.org/fusiondirectory-current/debian-stretch stretch main
deb http://repos.fusiondirectory.org/fusiondirectory-extra/debian-stretch stretch main
```

**Comment by liszca** [comment]
Was missing "php7.3-xml" now it does work, please add it to the checks during installation.

**Comment by kadogo** [comment]
Hello @liszca 

Which are your OS?

Did you installed any plugins?

Do you reproduce the same issue without ticking "Enforce encrypted connections"?

It look like the release version not have explicite dependence to php-xml it's fixed in 1.3-fixes

*  https://gitlab.fusiondirectory.org/debian/fusiondirectory/blob/master/debian/control
*  https://gitlab.fusiondirectory.org/debian/fusiondirectory/blob/1.3-fixes/debian/control#L48

Cheers
